flink-cdc-common/src/test/java/org/apache/flink/cdc/common/utils/SchemaUtilsTest.java

```diff
@@ -27,20 +27,21 @@
 import org.apache.flink.cdc.common.types.DataType;
 import org.apache.flink.cdc.common.types.DataTypes;
 
-import org.junit.Assert;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
It change SchemaUtilsTest IT to Junit5, it seems not related to this PR. And we separate a PR to do it in usually.

Reply:
OK, I will be separate a new PR.

Reply:
Rollback.